location/lbs/activity/audioset/dataprocessing/audio_processing.py

```diff
@@ -0,0 +1,466 @@
+"""
+Script to process a csv file from AudioSet into a csv file with features.
+
+Each row of the inputted csv is parsed for the video_id, start_time_seconds,
+end_time_seconds, and labels
+The downloaded audios would be clipped into the labelled 10 seconds segments
+The audio segments are then processed using Librosa to extract features
+
+Usage
+Standalone script:
+python audio_processing.py --src_dir PATH_TO_INPUT_FILES --des_dir DEST_DIR PATH_TO_OUTPUT_FILES_--audioset_csv CSV FILE --labels LABELS --redo BOOLEAN TO REDOWNLOAD THE YOUTUBE FILES
+example:
+python audio_processing.py --src_dir location/lbs/activity/audioset/dataprocessing/example_src_dir --des_dir location/lbs/activity/audioset/dataprocessing/example_dest_dir_--audioset_csv balanced_train_segments --labels "Gunshot, gunfire" --False
+
+Build with bazel:
+bazel build location/lbs/activity/audioset/dataprocessing:audio_processing
+bazel-bin/location/lbs/activity/audioset/dataprocessing/audio_processing --src_dir location/lbs/activity/audioset/dataprocessing/example_src_dir --des_dir location/lbs/activity/audioset/dataprocessing/example_dest_dir_--audioset_csv balanced_train_segments --labels "Gunshot, gunfire" --False
+
+
+--src_dir is the path to the directory containing the input files
+(audioset csv file ontology.json)
+--dest_dir is the path to the directory containing the output files
+(output csv file)
+--labels included in the AudioSet dataset can be used (i.e "Gunshot, gunfire")
```

Comment:
this can be merged into the flags section (otherwise, it looks redundant)

Reply:
Ok. I'll do that. Thank you.